docs/changes.xml

```diff
@@ -63,6 +63,13 @@ object, notably with Falcon.
 </para>
 </change>
 
+<change type="bugfix">
+<para>
+an issue in Clang 3.8 and later made the ruby module fail to configure
+in some systems.
+</para>
+</change>
```

Comment:
It doesn't look like a Unit bug at all, so listing it as a "bugfix" seems wrong. As a feature, it looks too insignificant (especially knowing that it can be workarounded in early versions too by manually adding of relevant flag). More like a sugar for package maintainers and enthusiasts who build from sources.

I suggest to not bloat changelog by such trivial improvement.